### ISSUE TITLE
Lemken Azurit 9 cables not working

### ISSUE THREAD

**Issue by krzar** [stijnwop/manualAttach#123]
I cannot connect cables on Lemken Azurit 9, there are connected visual but I can't unfold the drill.

**Comment by HB-2012** [comment]
I can confirm this issue, I'm dealing with the same problem. I've confirmed on a new (and different) map with 0 mods except ManualAttach. Using the Azurit 9 with ManualAttach is rather buggy. The Azurit 9 has a separate fertilizer box on the front (Solitair 23). Mounting that one and connecting the hoses before you mount the planter will result in the entire unit working properly. You might have to try mounting and unmounting a few times (that's what I did, anyway).

Without this mod the Azurit 9 can be used without fertilizer box. The [Lemken website](https://lemken.com/de/aussaat/pneumatische-drillmaschinen/) also shows it as an optional module, so I believe Manual Attach shouldn't change this behaviour. The Kuhn seeder combination doesn't have this issue: the drill can be unfolded and attached properly without the front unit.

I hope this helps.
